daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/GrpcLedgerClient.scala

```diff
@@ -382,7 +382,16 @@ class GrpcLedgerClient(val grpcClient: LedgerClient, val applicationId: Applicat
       grpcClient.userManagementClient
         .listUsers(pageToken = pageToken, pageSize = pageSize)
         .flatMap { case (users, nextPageToken) =>
-          if (users.size < pageSize) Future.successful(users.toList)
+          // A note on loop termination:
+
+          // We terminate the loop when the nextPageToken is empty.  Currently the API always
```

Comment:
The fix has been merged so I think we can drop this now.

Reply:
yup